filebeat/tests/system/test_modules.py

```diff
@@ -165,8 +165,8 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
             # the event was ingested to Elasticsearch
             assert "ingested" in obj["event"], "missing event.ingested timestamp"
 
-            assert "error" not in obj, "not error expected but got: {}".format(
-                obj)
+            assert "error" not in obj, "not error expected but got: {}.\n The related error message is: {}".format(
+                obj, obj["error"]["message"])
```

Comment:
nit (since we know that `error` key exists but `message` can be missing which will give an exception)
```suggestion
                obj, obj["error"].get("message")
```